### ISSUE TITLE
Licensing of talks

### ISSUE THREAD

**Issue by awvwgk** [fortran-lang/talks#22]
To make the material here reusable we should encourage to publish all material added here under an open source license.

I wonder what the best choice here is? Disclaiming copyright for graphics (CC0 or Unlicense) could be a useful strategy for easing adaption in a different context. For most material probably a permissive license (MIT, BSD-3-Clause, CC-BY) would be useful. Custom themes, especially for beamer/LaTeX presentation themes this might require GPL and LaTeX project licenses to be used. In summary a single presentation might contain several different licenses for the individual components.

Maybe a note in the README should give some advice on this complications, *i.e.* that the presentations here are not automatically free to use, but potentially require permissions first.

**Comment by milancurcic** [comment]
I agree. We can recommend CC-BY in the README, and people can use another license on an individual basis. For example in the [FortranCon2020-community](https://github.com/fortran-lang/talks/tree/main/FortranCon2020-community) talk, we specified CC-BY.

Notice that when we include presentation framework code like e.g. reveal.js, that code still needs to be under the original license.
